virttest/png_utils.py

```
@@ -0,0 +1,57 @@
+            compression = struct.unpack("B", fin.read(1))[0]
+            filter_method = struct.unpack("B", fin.read(1))[0]
+            interlace = struct.unpack("B", fin.read(1))[0]
+
+            # Validate PNG header values
+            assert bit_depth in [1, 2, 4, 8, 16]
+            assert color_type in [0, 2, 3, 4, 6]
+            assert compression == 0  # Only compression method 0 is defined
+            assert filter_method == 0  # Only filter method 0 is defined
+            assert interlace in [0, 1]  # 0 = no interlace, 1 = Adam7 interlace
+
+        return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)